verl/workers/rollout/vllm_rollout/vllm_rollout_spmd.py

```diff
@@ -87,7 +87,11 @@ def __init__(self, model_path: str, config: DictConfig, tokenizer, model_hf_conf
         """
         super().__init__()
         self.config = config
-        assert not (not config.enforce_eager and config.free_cache_engine), "disable CUDA graph (enforce_eager = False) if free cache engine"
+        if vllm_version in (
```

Comment:
nit: better to keep the versions in verl/workers/rollout/vllm_rollout/ as a constant and refer to this constant in all these places instead (or have a util function for version check).